test/test_cli.py

```diff
@@ -0,0 +1,83 @@
+import shutil
+import pytest
+import subprocess
+import os
+from app import application
+import time
+
+
+@pytest.mark.main
+def test_initialize_repo():
+    try:
+        shutil.rmtree("bitcointranscripts", ignore_errors=True)
+    except Exception as e:
+        assert False
+
+
+@pytest.mark.main
+def test_find_source_type():
+    video = application.check_source_type("B0HW_sJ503Y") == "video"
+
+    playlist = application.check_source_type("PLPQwGV1aLnTuN6kdNWlElfr2tzigB9Nnj") == "playlist"
+
+    audio = application.check_source_type(
+        "https://anchor.fm/s/12fe0620/podcast/play/32260353/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging"
+        "%2F2021-3-26%2Fdc6f12e7-a547-d872-6ef6-7acfe755a692.mp3") == "audio"
+    assert audio and playlist and video
+
+
+def test_download_audio_file():
+    audio = application.get_audio_file(
+        "https://anchor.fm/s/12fe0620/podcast/play/32260353/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging"
+        "%2F2021-3-26%2Fdc6f12e7-a547-d872-6ef6-7acfe755a692.mp3", "test")
+    assert os.path.isfile(audio)
+    os.remove(audio)
+
+
+@pytest.mark.main
+def test_process_mp3():
+    result = application.process_mp3("testAssets/audio.mp3", "tiny")
+    with open("testAssets/transcript.txt", "r") as file:
+        data = str(file.read().strip("\n"))
+        result = result.strip(" ")
+        assert data == result
```

Comment:
Ran 3 times, could not get this to pass. Additionally, it takes about 95% of the time (~90 sec) to run this test. This should be pulled for now and rethought.

<img width="1212" alt="Screen Shot 2023-02-02 at 1 57 41 PM" src="https://user-images.githubusercontent.com/755825/216423654-afa26008-2abd-4a58-a2b8-8873cfc51716.png">

Reply:
I'm infact scrapping this test as I don't think that part needs to be tested.